backend/src/main/kotlin/org/loculus/backend/service/submission/UploadDatabaseService.kt

```diff
@@ -228,7 +236,11 @@ class UploadDatabaseService(
             LEFT JOIN
                 sequence_upload_aux_table
                 ON metadata_upload_aux_table.upload_id = sequence_upload_aux_table.upload_id 
-                AND metadata_upload_aux_table.submission_id = sequence_upload_aux_table.submission_id
+                AND EXISTS (
```

Comment:
something is wrong with this function - when I submit multi-segmented sequences the unalignedNucleotideSequences object is empty?

Reply:
oh no this is actually working fine! its just that issue with revising on the page and the website searching for exact segment names in the original data map